css/style.css

```diff
@@ -0,0 +1,342 @@
+/*
+Project Name: Transition to Employment Toolkit
+Client: Transition to Employment
+Author: Hyun Masiello 
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+.clearfix:before,
+.clearfix:after {
+  content: "";
+  display: table;
+}
+
+.clearfix:after {
+  clear: both;
+}
+
+.clearfix {
+  zoom: 1; /* ie 6/7 */
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+/*
+colors: 
+
+*/
```

Comment:
add a `html` selector where you set up your default `font-family`, `font-size`, and `color`.

e.g.

```css
html {
 font-family: Arial;
 font-size: 14px;
 line-height: 1.4;
 color: #333;
}
```